algorithm/searching-algo-python/interpolation_search.py

```diff
@@ -0,0 +1,40 @@
+print("Interpolation Search using Python")
+
+str = input("Please enter a list of sorted numbers (in ascending order) separated by spaces: ")
```

Comment:
```suggestion
inout_string= input("Please enter a list of sorted numbers (in ascending order) separated by spaces: ")
```
Let's avoid using the name str for a variable, as it shadows the built-in str function in Python.